sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -111,16 +111,23 @@ export interface CustomFormModelField {
 export interface CustomFormModelInfo {
     modelId: string;
     modelName?: string;
+    properties?: CustomFormModelProperties;
```

Comment:
so this means it is nullable? in which cases will properties be nullable?

Reply:
The scenario I can think of is for unlabeled where properties are not returned. If this is the case, in other languages we decided to still assigned value to properties and to `isComposedModel` of false.

Reply:
Both `attributes` and `isComposed` are optional in the generated layer and not required in the spec. I'm not opposed to coercing them to defined values in certain situations, but leaving them optional in the type system is the safest option. Easy to go from optional to strict but not the other way.

Reply:
From swagger I understand, as the service doesn't return it from unlabeled models. But in this specific case, a customer doesn't need to know that as we can calculate it in the SDK side. So why expose it?

>I'm not opposed to coercing them to defined values in certain situations

Could you expand more in which situations will making it strict will make sense, and how this one is not?

Reply:
Also, this is not a blocking PR issue as this is beta.1
But for future releases, it will be good to understand this more and make sure it is ok for JS to be different than all other languages here

Reply:
If we are sure that a model is either composed or not (there's no unknown state) then setting it to false feels better than implicit missing-is-false.